src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -287,7 +358,64 @@ export default {
 }
 </script>
 
+<style>
+.vs__dropdown-menu {
+	min-width: 95px !important;
+}
+</style>
+
 <style lang="scss" scoped>
+
+:deep(.vs__clear) {
+	display: none;
+}
+
+:deep(.v-select) {
+	min-width: 95px !important;
+	.vs__dropdown-toggle {
+		background: none;
+	}
+}
+
+.selected-page{
+	padding-left: 5px;
+
+	display:inline-flex;
+	align-items: center;
+}
+
+.page-number{
+	padding-inline: 5px;
+}
+
+.large-width{
+	width: 100vw !important;
+	left: 0 !important;
+}
+
+.pagination-items{
+	background-color: var(--color-main-background);
+	border-radius: var(--border-radius-large);
+	pointer-events: all;
+
+	display: flex;
+	align-items: center;
+}
+
+.pagination-footer{
+	position:absolute;
+	box-shadow: var(--box-shadow);
+	filter: drop-shadow(0 1px 10px var(--color-box-shadow));
+	bottom: 10px;
```

Comment:
Bottom spacing could be a bit increased to avoid the drop shadow conflicting with the scrollbar from my perspective but nit picking:


<img width="987" alt="Screenshot 2024-05-03 at 14 47 46" src="https://github.com/nextcloud/tables/assets/3404133/b3b16282-7922-4efb-a464-9de5ed49c63d">

vs.

<img width="986" alt="Screenshot 2024-05-03 at 14 48 16" src="https://github.com/nextcloud/tables/assets/3404133/2d7fc10b-09dc-428b-82ed-fc5770bcf8d2">